internal/runtime/client/client.go

```diff
@@ -221,12 +229,15 @@ func (c *client) CallAllExtensions(ctx context.Context, hook runtimecatalog.Hook
 		}
 		// If the object namespace isn't matched by the registration NamespaceSelector skip the call.
 		if !namespaceMatches {
+			log.V(5).Info(fmt.Sprintf("skipping object %s/%s as it does not match selector %q of ExtensionConfig", forObject.GetNamespace(), forObject.GetName(), registration.NamespaceSelector))
```

Comment:
I think this log is misleading. We're not skipping the entire object (which would be a specific cluster), we're skipping a specific extension handler for it because it's ExtensionConfig namespace selector doesn't match

Reply:
Oops. You are right. I will correct that.